GetFitLA_v3/app/src/main/java/practicum/getfitla_v3/List_Adaptor.java

```diff
@@ -0,0 +1,77 @@
+package practicum.getfitla_v3;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.util.List;
+
+
```

Comment:
Eventually, you are going to need an adapter class for every activity that implements a RecyclerView of ListView type of display. This is because each adapter organizes the data specific to the activity.